spec/site_github_munger_spec.rb

```diff
@@ -231,4 +231,27 @@
       end.to raise_error(Jekyll::GitHubMetadata::Client::BadCredentialsError)
     end
   end
+
+  context "render the 'uninject' fixture test site" do
+    let(:source) { File.expand_path("test-site-uninject", __dir__) }
+    let(:dest) { File.expand_path("../tmp/test-site-uninject-build", __dir__) }
+
+    it "process site twice (simulate reset), check API calls" do
```

Comment:
Might be worth ensuring that the fields are properly rendered in the resulting HTML after the second `site.process` call?